### PR TITLE
costa: remove dependecies when uninstall plugins

### PR DESCRIPTION
**Pull request by FeelyChau**

### CODE REVIEW

**Comment by yorkie** [comment]
The test is needed :)

**Comment by FeelyChau** [comment]
> The test is needed :)

Done.

File: packages/costa/test/runtime_test.ts [hunk 1]
```diff
@@ -1,55 +1,88 @@
...
+    await costa.install(npmPkg, process);
+    // make sure js packages are installed.
+    await stat(path.join(costa.options.installDir, 'node_modules', npmPkg.name));
     // make sure python caches are used.
     await stat(path.join(costa.options.installDir, '.pip/selfcheck.json'));
   });
 
   it('should fetch a plugin and install from tarball', async () => {
-    const collectCsvWithSpecificVer = await costa.fetch('https://registry.npmjs.org/@pipcook/plugins-csv-data-collect/-/plugins-csv-data-collect-0.5.8.tgz');
+    try {
```
Comment (yorkie) [inline packages/costa/test/runtime_test.ts:73]: This needs to be reverted.

File: packages/costa/test/runtime_test.ts [hunk 2]
```diff
@@ -69,48 +102,24 @@ describe('create a costa runtime', () => {
     expect(collectCsvOnBare.version).toBe(collectCsvLatest.version);
   });
 
-  it('should install the package without conda packages', async () => {
```
Comment (yorkie) [inline packages/costa/test/runtime_test.ts:72]: What's the reason to remove this case?

File: packages/costa/test/runtime_test.ts [hunk 3]
```diff
@@ -69,48 +102,24 @@ describe('create a costa runtime', () => {
...
-    await costa.install(collectCsv, process);
-    await stat(path.join(
-      costa.options.installDir,
-      'node_modules',
-      collectCsv.name
-    ));
-  });
-
-  it('should install the package with conda packages', async () => {
```
Comment (yorkie) [inline packages/costa/test/runtime_test.ts:81]: Ditto, we need a reason to remove this case.

File: packages/costa/test/runtime_test.ts (hunk 2 above)
Comment (FeelyChau) [reply packages/costa/test/runtime_test.ts:72]: https://github.com/alibaba/pipcook/pull/518/files#diff-bdea6cb8b04e910bac8851b60725bbaeL17
This case has tested `install the package without conda packages`.

File: packages/costa/test/runtime_test.ts (hunk 3 above)
Comment (FeelyChau) [reply packages/costa/test/runtime_test.ts:81]: https://github.com/alibaba/pipcook/pull/518/files#diff-bdea6cb8b04e910bac8851b60725bbaeL30

File: packages/costa/test/runtime_test.ts (hunk 1 above)
Comment (FeelyChau) [reply packages/costa/test/runtime_test.ts:73]: Updated.

**Review by yorkie — APPROVED** [review]
Looks good to me

**Pull request merged**